components/docs-chef-io/content/automate/ha_remove_single_node_from_cluster.md

```diff
@@ -63,6 +63,20 @@ The command requires some arguments to determine which types of nodes you want t
     chef-automate node remove --postgresql-ip "<postgresql-ip-address>"
     ```
 
+You can remove other unreachable nodes while removing a node
+
+- To remove unreachable nodes while removing a node, pass the flag `--remove-unreachable-node` or `-r` like example below
```

Comment:
Replace it with: - To remove unreachable nodes while removing a node, pass the flag `--remove-unreachable-node` or `-r` like the example below